src/lib/ceramic/CeramicFeed.js

```diff
@@ -27,6 +27,11 @@ class CeramicFeed {
 
   async getPost(postId: string) {
     const post = await Post.find(postId)
+
+    if (post?.content?.tags?.[0]?.slug === 'publishDapp') {
```

Comment:
this is not DRY
it should be changed to if(!serialized) return

Reply:
done